src/buildConfig.js

```diff
@@ -8,15 +8,88 @@ import {
   DEFAULT_SHOW,
 } from './const';
 
+/**
+ * Starting from the given index, increment the index until an array element with a "value" property is found
+ * @param {Array} stops
+ * @param {number} startIndex
+ * @returns {number}
+ */
+const findFirstValuedIndex = (stops, startIndex) => {
+  for (let i = startIndex, l = stops.length; i < l; i += 1) {
+    if (stops[i].value != null) {
+      return i;
+    }
+  }
+  throw new Error('Could not find right-nearest valued stop');
```

Comment:
I guess this shouldn't really happen but I think it would be good to elaborate further and indicate that this is related to the color threshold option, since these errors are presented to the user in the error card.